ca_description/urdf/create_light_sensor.xacro

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+<xacro:macro name="light_sensor" params="prefix pitch reflect">
+
+    <xacro:include filename="$(find ca_description)/urdf/common_properties.xacro"/>
+    
+    <xacro:property name="name" value="${prefix}_light_sensor"/>
+    <xacro:property name="parent_link" value="base_link"/>
+    <xacro:property name="x" value="0.095"/>
```

Comment:
I recommend you to pass the whole origin block as a parameter. See [here](http://wiki.ros.org/xacro#Macros) (*origin element).